composer.json

```diff
@@ -1,18 +1,15 @@
 {
   "name": "pluggit/monitoring",
   "description": "Monitoring abstraction layer",
+
   "require": {
-    "php": "^5.5|^7.0",
-    "psr/log": "^1.0"
+    "php": "^8.1",
+    "psr/log": "^3.0"
   },
   "require-dev": {
-    "phpspec/phpspec": "^2.5",
-    "behat/behat": "^3.1",
-    "phpunit/phpunit": "4.8.*",
+    "phpspec/phpspec": "^7.2.0",
     "squizlabs/php_codesniffer": "2.*",
-    "henrikbjorn/phpspec-code-coverage": "2.*",
-    "phpunit/php-code-coverage": "^2.2",
```

Comment:
The henrikbjorn/phpspec-code-coverage package was abandoned. And the extension was never enabled in the phpsec.yml in the first place, so I remove it.

Regarding the phpunit/php-code-coverage, this is again a dependency of a dependency, not used in our own code.